### ISSUE TITLE
DeprecationWarning with Python >=3.2

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1312]
When warnings are enabled, then 1 DeprecationWarning occurs in test suite. I use Sphinx trunk (currently revision bcf6b5bbafc7). I attach patch.

Results of test suite with Python 3.3:

```
................................................................................................................................................................................................................................................/usr/lib64/python3.3/site-packages/nose/case.py:198: DeprecationWarning: Please use assertEqual instead.
  self.test(*self.arg)
........................SS........................
----------------------------------------------------------------------
Ran 290 tests in 132.705s

OK (SKIP=2)
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1312
- Originally reported by: Arfrever Frehtes Taifersar Arahesis
- Originally created at: 2013-11-18T14:18:31.640

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-10 14:10:54+00:00_

Thanks for the patch!

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-10 14:11:05+00:00_

Closes #1312: use non-deprecated unittest-like names

→ <<cset 82e294ec0f427c936f137e66cca2a27b2510bdc4>>
